website/content/docs/bloop-basics.md

```diff
@@ -0,0 +1,127 @@
++++
+toc = true
+weight = 2
+draft = false
+title = "The basics of bloop"
+date = "2018-02-09T10:15:00+01:00"
+description = "What is bloop and which problems does it solve?"
+bref = "Learn the ideas behind bloop, which problems solve and how it's designed."
++++
+
+## An overview of Bloop
+
+Bloop is a Scala command-line tool and build server developed at the Scala
+Center to make the compile and test feedback loop as tight as possible. This
+means a faster and more productive developer workflow.
+
+### A common pain point
+
+Edit, compile and test workflows are the bread and butter of our daily jobs.
+When our build is slow to respond, our productivity drops. Can Scala get as
+tight edit/compile/test workflows as in Clojure or OCaml? Bloop is a
+command-line tool and build server that brings you a tighter developer
```

Comment:
faster instead of tighter to stop repeating it?

Reply:
I prefer to repeat it because they don't mean quite the same, and we're just using it in the comparative form here compared to the previous sentence. Sounds good to me.